src/main/java/duke/command/EventCommand.java

```diff
@@ -0,0 +1,61 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.Storage;
+import duke.Ui;
+import duke.exceptions.InvalidFileException;
+import duke.exceptions.InvalidInputException;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+/**
+ * EventCommand class to execute command that add an event into
+ * the TaskList.
+ */
+public class EventCommand extends Command {
+    public static final String MESSAGE_SUCCESS = "Got it. I've added this task:\n";
+    public static final String MESSAGE_PARSE_ERROR = "Invalid date and time format.\n"
+            + "Please enter date and time in the format: yyyy-MM-dd HH:mm";
+
+    public EventCommand(String input) {
+        super(input);
+    }
+
+    /**
+     * Execute the command to add an event into the task list.
```

Comment:
Start the javadoc with Executes (verb with s)
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)